app/views/components/item-card/subitems.js

```diff
@@ -32,10 +35,13 @@ var SubItem = React.createClass({
     ev.preventDefault();
     let node = this.refs.addItemInput.getDOMNode();
     let title = node.value;
-    ItemActions.addItem(this.props.productId, {
+    let productId = this.getParams().id;
+    let storyId = this.getParams().number;
```

Comment:
why pull these off of `this.getParams()` (which I'm pretty sure is getting deprecated in the next version of react-router) instead of using the props?